### PR TITLE
📖 Docs general improvements 📖

### PR DESCRIPTION
**Pull request by tapaswenipathak**
:sunny: Fix https://github.com/readthedocs/readthedocs.org/issues/5771. :sunny:

- Improvement in titles
- Remove/add any extra docs
- Docs splitting and adding index files

### CODE REVIEW

File: docs/builds.rst [hunk 1]
```diff
@@ -172,6 +172,104 @@ If you're having trouble getting your version to build, try wiping out the exist
 Build environment
 -----------------
 
+Build environments use `Docker`_ to handle container virtualization. To perform
```
[the remark this answers is not in the document]
Comment (tapaswenipathak) [reply docs/builds.rst]: The ticket says I should add build and build environment docs in 1 doc, should I separate build environment doc?

[the remark this answers is not in the document]
Comment (tapaswenipathak) [reply docs/builds.rst]: Sure.

[the remark this answers is not in the document]
Comment (tapaswenipathak) [reply docs/builds.rst]: Ah, now that's a problem a little more work, I will check that and push a commit.

**Comment by tapaswenipathak** [comment]
@stsewd: done.

**Comment by stsewd** [comment]
@tapaswenipathak there is an error on the CI server, make sure your branch is up to date with master.

**Comment by tapaswenipathak** [comment]
On Fri, 6 Sep 2019, 00:02 Santos Gallegos, <notifications@github.com> wrote:

> @tapaswenipathak <https://github.com/tapaswenipathak> there is an error
> on the CI server, make sure your branch is up to date with master.
>

I can remove that completely np. I will let you know.

**Comment by tapaswenipathak** [comment]
@stsewd: Inline commit removes that.

**Comment by stsewd** [comment]
@tapaswenipathak CI is still failing, you can check how to run tests locally here https://docs.readthedocs.io/en/stable/development/tests.html and fix them.

**Comment by tapaswenipathak** [comment]
Just a note: weirdly, the path is having issues. I don't get them when I run in local, fixing.

**Comment by tapaswenipathak** [comment]
@stsewd: fixed Travis, review ping.

**Comment by stsewd** [comment]
Sorry, haven't time to re-review this. Still valid.

**Pull request closed without merging**